### PR TITLE
support loadBalancerIP in helm chart for service

### PR DESCRIPTION
**Pull request by FlipEnergy**
Signed-off-by: Dennis Zhang <dennis.zhang.nrg@gmail.com>

For issue #5282

### CODE REVIEW

**Comment by FlipEnergy** [comment]
I signed it

**Comment by desaintmartin** [comment]
Thanks!
Since this is a new feature, could you bump a minor version instead of patch?
Could you document the new parameter in the README?

**Comment by FlipEnergy** [comment]
@desaintmartin Ready for your review. Thanks

**Comment by desaintmartin** [comment]
Sorry, I would like the new CI to get merged before merging new PRs, since it will catch a lot of potential errors.

**Comment by desaintmartin** [comment]
Could you rebase again? CI just got merged.

File: aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v1
 name: kubernetes-dashboard
-version: 2.1.1
+version: 2.2.1
```
Comment (desaintmartin) [inline aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml:17]:
```suggestion
version: 2.2.0
```

Comment (FlipEnergy) [reply aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml]: /facepalm myself

**Comment by desaintmartin** [comment]
Thanks!
/lgtm

**Pull request merged**